package.json

```diff
@@ -54,41 +54,41 @@
     ]
   },
   "dependencies": {
-    "@arcgis/core": "^4.25.5",
+    "@arcgis/core": "^4.26.5",
     "@fortawesome/free-solid-svg-icons": "^6.3.0",
     "@fortawesome/react-fontawesome": "^0.2.0",
     "@react-hook/window-size": "^3.1.1",
     "bootstrap": "^5.2.3",
     "clsx": "^1.2.1",
-    "downshift": "^7.2.0",
-    "eslint-config-prettier": "^8.6.0",
-    "jquery": "^3.6.3",
+    "downshift": "^7.6.0",
+    "eslint-config-prettier": "^8.7.0",
+    "jquery": "^3.6.4",
```

Comment:
what's this used for?

Reply:
Good question! I don't know. I removed it and can't find anything wrong. I also searched the code base and can't find where I was using it. Perhaps a relic from the past.